packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -171,21 +137,31 @@ async function writeToPackageJson (packageJsonPath: string, uploadUrl?: string,
     // Default to two spaces if indent cannot be detected
     const existingIndent = detectIndent(data).indent || '  '
 
-    let uploadCommand = 'bugsnag-cli upload react-native-android'
-    let buildCommand = 'bugsnag-cli create-build'
+    let createBuildCommand = 'bugsnag-cli create-build'
+    let rnAndroidUploadCommand = 'bugsnag-cli upload react-native-android'
+    let androidUploadCommand = 'bugsnag-cli upload android'
+    let rnIosUploadCommand = 'bugsnag-cli upload react-native-ios'
+    let dsymUploadCommand = 'bugsnag-cli upload dsym'
 
     if (uploadUrl) {
-      uploadCommand += ` --upload-api-root-url=${uploadUrl}`
+      rnAndroidUploadCommand += ` --upload-api-root-url=${uploadUrl}`
+      androidUploadCommand += ` --upload-api-root-url=${uploadUrl}`
+      rnIosUploadCommand += ` --upload-api-root-url=${uploadUrl}`
+      dsymUploadCommand += ` --upload-api-root-url=${uploadUrl}`
     }
 
     if (buildUrl) {
-      buildCommand += ` --build-api-root-url=${buildUrl}`
+      createBuildCommand += ` --build-api-root-url=${buildUrl}`
     }
 
     packageJson.scripts = {
       ...packageJson.scripts,
-      'bugsnag:create-build': buildCommand,
-      'bugsnag:upload-android': uploadCommand
+      'bugsnag:create-build': createBuildCommand,
+      'bugsnag:upload-android': androidUploadCommand,
+      'bugsnag:upload-rn-android': rnAndroidUploadCommand,
+      'bugsnag:upload-dsym': dsymUploadCommand,
+      'bugsnag:upload-rn-ios': rnIosUploadCommand,
+      'bugsnag:upload': 'npm run bugsnag:upload-android && npm run bugsnag:upload-rn-android && npm run bugsnag:upload-rn-ios && npm run bugsnag:upload-dsym'
```

Comment:
As discussed IRL, let's call the CLI directly here to avoid yarn compatibility issues.